app/src/androidTest/java/ru/tinkoff/favouritepersons/tests/EditStudentPageTests.kt

```diff
@@ -0,0 +1,132 @@
+package ru.tinkoff.favouritepersons.tests
+
+import io.qameta.allure.kotlin.AllureId
+import io.qameta.allure.kotlin.junit4.DisplayName
+import org.junit.Test
+import ru.tinkoff.favouritepersons.PersonDescription
+import ru.tinkoff.favouritepersons.pages.EditStudentPage
+import ru.tinkoff.favouritepersons.pages.MainPage
+
+class EditStudentPageTests : BaseTest() {
+
+    @Test
+    @AllureId("5")
+    @DisplayName("Проверка открытия второго экрана с данными пользователя")
+    fun checkOpeningSecondScreenWithUserData() = run {
+        mock(listOf(PersonDescription()))
+        MainPage {
+            clickButtonAddPerson()
+            clickButtonNetwork()
+            clickPersonInPosition(0)
+        }
+        EditStudentPage {
+            checkNameHasText("Leo")
+            checkSurnameHasText("Lampo")
+            checkGenderHasText("М")
+            checkBirthdayHasText("1951-08-01")
+        }
+    }
```

Comment:
Старайся по возможности как можно меньше констант ни к чему не привязанных использовать предпочитая в начале теста заводить expected значения.
```suggestion
        val expectedPerson = PersonDescription()
        mock(listOf(expectedPerson))
        MainPage {
            clickButtonAddPerson()
            clickButtonNetwork()
            clickPersonInPosition(0)
        }
        EditStudentPage {
            checkNameHasText(expectedPerson.name)
            checkSurnameHasText(expectedPerson.lastname)
            checkGenderHasText(expectedPerson.gender)
            checkBirthdayHasText(expectedPerson.date)
        }
    }
```